### DISCUSSION TITLE
4.8- OCI - Deploy/Core/OfflineInstaller

### DISCUSSION THREAD

**Discussion by ddevadat** [Sunbird-Ed/Community#631] [Bugs] [answered]
For 4.8 upgrde
https://github.com/project-sunbird/sunbird.org-docs/blob/4.8.0/developer-docs/upgrade/index.md

We couldnt find the jenkins jobs for Deploy/Core/OfflineInstaller
We took the job definition from release-5.1.0 and tried to deploy.

But we are getting this exception

```
TASK [desktop-deploy : Copy the desktop assests folder from inside the private repo location to root location.] ***
fatal: [localhost]: FAILED! => {
    "changed": true,
    "cmd": "cp -r /var/lib/jenkins/workspace/Deploy/dev/Kubernetes/OfflineInstaller/private/desktop-assets /var/lib/jenkins/workspace/Deploy/dev/Kubernetes/OfflineInstaller/",
    "delta": "0:00:00.073634",
    "end": "2023-09-27 14:12:36.507367",
    "rc": 1,
    "start": "2023-09-27 14:12:36.433733"
}

STDERR:

cp: cannot stat '/var/lib/jenkins/workspace/Deploy/dev/Kubernetes/OfflineInstaller/private/desktop-assets': No such file or directory


MSG:

non-zero return code
```

is this deployment really needed? because we didnt this in the release-5.1.0? If needed for 4.8 , could you please help with the error

**Comment by gandham-santhosh** [comment, marked as answer]
@ddevadat This is the deploy job for the offline desktop app. You don't need this if you are using Web Portal and Mobile to access the application

**Reply by ddevadat** [reply]
thanks for the clarification
